codeforpdx/PASS
# 586 feature request add sign up with existing pod option to the sign up page

## This PR:
Created the register with existing pod form on the signup page which accepts the URL to a pod provider. 


## Test steps

1. Create a new pod with https://opencommons.net/
2. On the Pass signup page, enter https://opencommons.net/ in the pod provider field in the register with existing pod form. 
3. Click the login to pod provider button. 
4. You should be redirected to https://opencommons.net/
5. Login and you should be redirected back to pass. Confirm that you are signed in.
